service/src/main/scala/org/apache/celeborn/server/common/http/HttpRequestHandler.scala

```diff
@@ -43,11 +47,11 @@ class HttpRequestHandler(
     val uri = req.uri()
     val msg = handleRequest(uri)
     val response = msg match {
-      case INVALID =>
+      case "invalid" =>
         if (prometheusHttpRequestHandler != null) {
           prometheusHttpRequestHandler.handleRequest(uri)
         } else {
-          s"invalid uri ${uri}"
+          s"invalid uri $uri"
```

Comment:
Could we add a help info here? We can do this in a separated pr after this pr.  I have raise a ticket https://issues.apache.org/jira/browse/CELEBORN-829

Reply:
@AngersZhuuuu, I will raise another pull request to follow above help info.